app/controllers/customers_controller.rb

```diff
@@ -1,5 +1,5 @@
 class CustomersController < ApplicationController
-
+  skip_before_filter :verify_authenticity_token
```

Comment:
I don't remember since it was so long ago but I think we need this to access omniauth env variable which is needed in case there are validation errors

Reply:
It is not necessary to put this line there and it is not safe. Please double check

Reply:
Okay it should work if we change it to "skip_before_filter :verify_authenticity_token, only: :create, :user_create" I think. I'll test removing it completely too but it shouldn't work. The OmniAuth callback url does not correctly verify the rails authenticity token and so will destroy any session data upon returning, thereby logging your current user out. So we do need to disable this for certain functions that are creating the user.